bin/oneoff/backfill_data/user_proficiencies

```diff
@@ -106,16 +106,26 @@
 require 'csv'
 require_relative '../../../dashboard/config/environment'
 
+# Fields:
+# start_date, end_date, end_timestamp, already_recorded
```

Comment:
:+1:

Reply:
I think there's one script ID missing here (303, the [express course](https://studio.code.org/s/express-2018)) It was script_id 258 in the previous version.

Reply:
Thanks for catching!